stage2/EXPORT_IMAGE

```diff
@@ -1 +1,4 @@
 IMG_SUFFIX="-lite"
+if [ $QEMU_MODE = "true" ]; then
```

Comment:
Quote marks around `$QEMU_MODE` for cases when it's not set at all

Reply:
I am testing something like:

> 	if [ $USE_QEMU -ne 1 ]; then

Reply:
Same problem. If `$USE_QEMU` is not set, then you get `bash: [: -ne: unary operator expected`